multicluster/cmd/multicluster-controller/memberclusterannounce_webhook.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Can you move webhook related codes into a standalone directory instead of `cmd`?

Reply:
Can you please suggest where to move it? I cannot be in controller/multicluster because it failed kubebuilder for some reason, so I moved it here.